ktor-shared/ktor-serialization/ktor-serialization-jackson/jvm/src/JacksonConverter.kt

```diff
@@ -94,6 +103,17 @@ public class JacksonConverter(
     }
 
     private companion object {
+        private val encodings = buildMap<String, JsonEncoding> {
+            JsonEncoding.entries.forEach { put(it.javaName, it) }
+            put("US-ASCII", JsonEncoding.UTF8) // Jackson automatically unescapes Unicode characters
+        }
```

Comment:
@osipxd I've implemented this feedback.

I Initially wanted to use the mapping to figure out all encodings that are supported by Jackson while doing serialization, but that would be rather useful to validate the charset even earlier, since only UTF-8, UTF-16, UTF-32 are valid.